pom.xml

```diff
@@ -847,13 +847,12 @@
                 </property>
             </activation>
             <modules>
-                <!-- <module>samples/server/petstore/erlang-server</module> note: make sample compilation work -->
                 <!-- clients -->
                 <!-- elm client not fully ready for CI yet
                 <module>samples/client/petstore/elm</module>-->
                 <module>samples/client/petstore/groovy</module>
                 <module>samples/client/petstore/rust</module>
-                <module>samples/client/petstore/perl</module>
+                <!--<module>samples/client/petstore/perl</module>-->
```

Comment:
Should this pom match the circlci poms? The go client is commented there and perl is commented here.

Reply:
Good question but they should not be the same as different CIs should test different clients/servers.